src/main/java/io/jenkins/plugins/audit/config/AuditLogConfiguration.java

```diff
@@ -0,0 +1,44 @@
+package io.jenkins.plugins.audit.config;
+
+import hudson.Extension;
+import jenkins.model.GlobalConfiguration;
+import org.apache.logging.log4j.core.LoggerContext;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.DataBoundSetter;
+
+import java.util.Objects;
+
+@Extension
+public class AuditLogConfiguration extends GlobalConfiguration {
+    private String logDestination;
+
+    public AuditLogConfiguration() {
+        super();
+        load();
+    }
+
+    public String getLogDestination() {
+        return logDestination;
+    }
+
+    @DataBoundSetter
+    public void setLogDestination(String logDestination) {
+        this.logDestination = logDestination;
+        save();
+        System.setProperty("auditFileName", logDestination);
+        LoggerContext.getContext().reconfigure();
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
(Minor coding style comment) There isn't a standard, but I think most people involved in Jenkins development prefer this style:
```
if (this == o) {
    return true;
} 
```